.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,85 @@
+name: Run tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
```

Comment:
Is this suppose to be changed before merge?
```suggestion
      fail-fast: true
```

Reply:
I would go with true, but in fact am OK with either.

Reply:
(hmm, wrong browser....)

Reply:
I would normally go with yes but actually the numpy dev job might cause the others to stop so maybe keep false until we discuss allowed failures etc?

Reply:
Yeah, you are right, the OSX job got canceled.

Reply:
since we already have multiple workflows, I'm getting on board of indeed separating the allowed failures one out. In a follow-up.